### PR TITLE
Added plugin support

### PR DESCRIPTION
**Pull request by martinheidegger**
Added plugin support (and updated dev dependencies)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,5 +40,27 @@
   "license": "MIT",
   "dependencies": {
     "boom": "^2.7.1"
-  }
+  },
```
Comment (alexgorbatchev) [inline package.json]: These shouldn't be included I think.

**Comment by martinheidegger** [comment]
@alexgorbatchev Fixed, sorry for the wrong data.

**Comment by alexgorbatchev** [comment]
Thanks for the PR. I renamed `ending` to `extension` and published as v2.1.0

**Comment by martinheidegger** [comment]
:+1+

**Pull request merged**